### DISCUSSION TITLE
Searching for a dimmer without fade effects

### DISCUSSION THREAD

**Discussion by Suxsem** [Koenkk/zigbee2mqtt#20419] [Devices]
Hi everyone!
I'm desperately searching for a dimmer module (that I can connect to a standard momentary wall switch) that DOES NOT have the fade-in effect. I hate when I enter the room, press the wall switch and have to wait a second or two for the light to be fully on.
Fade-out is ok (effect when the light is turned off).
Any Tuya or similar modules with this simple requirement?

**Comment by chris-1243** [comment]
I have one from sunricher (ZG9040A) which is not used anymore. I'm gonna have a try and let you know.

Have you tried to have "transition: 0" in your automation if the device support it ?

**Reply by Suxsem** [reply]
Yes but I'm interested in not having the fade effect when the light is turned on by physically pressing the wall switch, not from zigbee2mqtt. Waiting for your test, thank you!

**Reply by chris-1243** [reply]
I just made some tries and there should be a video included to my answer...

I would say there is a small delay. I'm not sure if it is what you are looking for.
The dimming is not the best in my opinion. I remember now why I changed them.

https://github.com/Koenkk/zigbee2mqtt/assets/142902565/aead66d8-cbe6-435d-b1e1-f726bd927069

**Reply by chris-1243** [reply]
When you open the device panel there is a tab "settings (specific)" where you should be able to modify the transition time...

I hope it helps

**Reply by Suxsem** [reply]
Thank you! It seems fast enough indeed but the camera auto exposure makes difficult to tell from here. Have you tried changing the transition time in zigbee2mqtt to zero in order to see if the transition is removed also when switched on/off by the physical button?
Can I ask you which dimmer are you using at the moment?

**Reply by chris-1243** [reply]
I tried to change the transition time from 0 to 3. Unfortunately, it had no real impact in my opinion. 

I do not use such dimmer for the moment. I kept my momentary switches which are connected to an ubisys S2 or a C4. The lights are then controlled via Zigbee. You could try the Ubisys D1 which is a dimmer and looks great on the paper and all the specific configuration you may have.
The module is thinner than all the modules I saw on the market.
The bad thing is the price...

[Ubisys D1](https://www.ubisys.de/en/products/light-and-consumers/universal-dimmer-d1/)

[D1 zigbee2mqtt](https://www.zigbee2mqtt.io/devices/D1.html)

**Reply by Suxsem** [reply]
It seems good indeed!!
Unfortunately I need 4 of them so for the moment I have to wait or my fiancé would kill me for the money spent.
Thank you for all the suggestions!
